src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -253,7 +253,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Friend Overrides Function GetImports() As ImmutableArray(Of String)
             ' TODO: implement (only called from VBI) https://github.com/dotnet/roslyn/issues/5854
-            Throw ExceptionUtilities.Unreachable
+            Dim importNames As New ArrayBuilder(Of String)(GlobalImports.Length)
```

Comment:
`Dim importNames = ArrayBuilder(Of String).GetInstance(GlobalImports.Length)`

Alternatively, use `SelectAsArray`.